packages/uma-auth-client/src/components/UmaConnectButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { ThemeProvider } from "@emotion/react";
+import { Button } from "@lightsparkdev/ui/components";
+import { themes } from "@lightsparkdev/ui/styles/themes";
+import { GlobalStyles } from "src/GlobalStyles";
+
+export const UmaConnectButton = () => {
+  const clientId = "1";
+  const redirectUri = "http://localhost:3000";
+  const responseType = "code";
+  const codeChallenge = "1234";
+  const codeChallengeMethod = "S256";
+
+  return (
+    <ThemeProvider theme={themes.umameDocsLight}>
```

Comment:
I know this is just a skeleton PR, but presumably this will be able to have a theme injected to style things, right?

Reply:
Yeah I still need to think through how the developer is going to pick their own theme to use here

Reply:
Added css variables in this follow up pr https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/105/Add-button-customizability-wrap-web-component-in-react-component